<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>文泉驿微米黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>DolorsChat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget{background-image: url(:/imgs/imgs/chatRoomBk.png);}
</string>
  </property>
  <widget class="QTextBrowser" name="messageBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#messageBrowser{background-color: rgba(255, 255, 255, 160);border:0px;
border-radius:3px;}


</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>372</y>
     <width>521</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>154</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{background-color: rgba(255, 255, 255, 180);}

QComboBox::drop-down {
  subcontrol-origin: padding;
  subcontrol-position: top right;
  width: 20px;

  border-left-width: 1px;
  border-left-color: darkgray;
  border-left-style: solid; /* just a single line */
  border-top-right-radius: 3px; /* same radius as the QComboBox */
  border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
  image: url(:/imgs/imgs/drop_down.png);
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeComboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>0</y>
      <width>38</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿微米黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{background-color: rgba(255, 255, 255, 180);}
QComboBox::drop-down {
  subcontrol-origin: padding;
  subcontrol-position: top right;
  width: 20px;

  border-left-width: 1px;
  border-left-color: darkgray;
  border-left-style: solid; /* just a single line */
  border-top-right-radius: 3px; /* same radius as the QComboBox */
  border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
  image: url(:/imgs/imgs/drop_down.png);
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
   </widget>
   <widget class="QToolButton" name="boldToolBtn">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>加粗</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/imgs/imgs/bo.ico</normaloff>:/imgs/imgs/bo.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="italicToolBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>倾斜</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/imgs/imgs/i.ico</normaloff>:/imgs/imgs/i.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="underlineToolBtn">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>下划线</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/imgs/imgs/u.ico</normaloff>:/imgs/imgs/u.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="colorToolBtn">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>更改字体颜色</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/imgs/imgs/color.ico</normaloff>:/imgs/imgs/color.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="sendToolBtn">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>传输文件</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/imgs/imgs/send.ico</normaloff>:/imgs/imgs/send.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="save">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>保存聊天记录</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/imgs/imgs/20140919013333834_easyicon_net_72.ico</normaloff>:/imgs/imgs/20140919013333834_easyicon_net_72.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="clearToolBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>清空聊天记录</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="imgs.qrc">
      <normaloff>:/imgs/imgs/clean.ico</normaloff>:/imgs/imgs/clean.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="messageTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>521</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#messageTextEdit{background-color: rgba(255, 255, 255, 160);;border:0px;
border-radius:3px;}</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="userNumLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>在线用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
border-radius:5px;
color:white;
background-color:rgb(172, 213, 191)</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>490</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
border-radius:5px;
color:white;
background-color:rgb(195, 195, 195);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="userTableWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>291</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#userTableWidget{background-color: rgba(255, 255, 255, 190);;border:0px;
border-radius:3px;}</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
    <property name="font">
     <font>
      <family>文泉驿微米黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>主机名</string>
    </property>
    <property name="font">
     <font>
      <family>文泉驿微米黑</family>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP地址</string>
    </property>
    <property name="font">
     <font>
      <family>文泉驿微米黑</family>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
